Makefile

```diff
@@ -102,7 +102,7 @@ endif
 	mkdir -p server/dist;
 ifneq ($(MM_SERVICESETTINGS_ENABLEDEVELOPER),)
 	@echo Building plugin only for $(DEFAULT_GOOS)-$(DEFAULT_GOARCH) because MM_SERVICESETTINGS_ENABLEDEVELOPER is enabled
-	cd server && env CGO_ENABLED=0 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-$(DEFAULT_GOOS)-$(DEFAULT_GOARCH);
+	cd server && env CGO_ENABLED=0 $(GO) build -ldflags '-X main.DebugBuild=true' $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-$(DEFAULT_GOOS)-$(DEFAULT_GOARCH);
```

Comment:
Is this just temporary or should this always be on?

Reply:
This will always be on if `MM_SERVICESETTINGS_ENABLEDEVELOPER` env variable is set at build time. In production build CI env, this variable will not be set and so the slash command won't work. I'll use this to tweak some more behaviours in production/dev builds.